llrgrwnvm007.in2p3.fr
NAME="AlmaLinux"
VERSION="9.3 (Shamrock Pampas Cat)"
ID="almalinux"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.3"
PLATFORM_ID="platform:el9"
PRETTY_NAME="AlmaLinux 9.3 (Shamrock Pampas Cat)"
ANSI_COLOR="0;34"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:almalinux:almalinux:9::baseos"
HOME_URL="https://almalinux.org/"
DOCUMENTATION_URL="https://wiki.almalinux.org/"
BUG_REPORT_URL="https://bugs.almalinux.org/"

ALMALINUX_MANTISBT_PROJECT="AlmaLinux-9"
ALMALINUX_MANTISBT_PROJECT_VERSION="9.3"
REDHAT_SUPPORT_PRODUCT="AlmaLinux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.3"
The Key4hep software stack is already set up, please start a new shell to avoid conflicts
Loading python/3.12.4
  Loading requirement: python/.3.12.4 torch/latest-312-cpu
key4hep ENV has been Set up
root version
/cvmfs/sw.hsf.org/key4hep/releases/2024-10-03/x86_64-almalinux9-gcc14.2.0-opt/root/6.32.04-vms5ij/bin/root
Added /cvmfs/sw.hsf.org/key4hep/releases/2025-05-29/x86_64-almalinux9-gcc14.2.0-opt/dd4hep/1.32-x2e5mf/lib to LD_LIBRARY_PATH
k4 local repo
Added /grid_mnt/vol_home/llr/ilc/shi/code/SiWECAL-Prototype/generation/run_scripts/Simu2025-06/build/. to the environment and removed any paths containing /build/
Some variables may have to be updated manually to point to the local installation
Traceback (most recent call last):
  File "/cvmfs/sw.hsf.org/key4hep/releases/2025-05-29/x86_64-almalinux9-gcc14.2.0-opt/dd4hep/1.32-x2e5mf/bin/ddsim", line 25, in <module>
    sys.exit(RUNNER.run())
             ^^^^^^^^^^^^
  File "/cvmfs/sw.hsf.org/key4hep/releases/2025-05-29/x86_64-almalinux9-gcc14.2.0-opt/dd4hep/1.32-x2e5mf/lib/python3.11/site-packages/DDSim/DD4hepSimulation.py", line 309, in run
    import ROOT
  File "/cvmfs/sw.hsf.org/key4hep/releases/2024-10-03/x86_64-almalinux9-gcc14.2.0-opt/root/6.32.04-vms5ij/lib/ROOT/__init__.py", line 25, in <module>
    import cppyy
  File "/cvmfs/sw.hsf.org/key4hep/releases/2024-10-03/x86_64-almalinux9-gcc14.2.0-opt/root/6.32.04-vms5ij/lib/cppyy/__init__.py", line 173, in <module>
    gbl.std.make_shared = make_smartptr(gbl.std.shared_ptr, gbl.std.make_shared)
                                                            ^^^^^^^^^^^^^^^^^^^
AttributeError: <class 'cppyy._cpython_cppyy.std'> has no attribute 'make_shared'. Full details:
  type object 'std' has no attribute 'make_shared'
  'make_shared' is not a known C++ class
  'make_shared' is not a known C++ template
  'make_shared' is not a known C++ enum
